cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.core.management import BaseCommand, CommandError
+from django.db import connections
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        connection = None
+        while not connection:
+            try:
+                connection = connections["default"]
```

Comment:
The connections dictionary will always contain a connection for 'default', whether or not the database is actually available. Trying to access connections['default'] will not raise a CommandError if the database is unavailable. Instead, you need to actually try using the connection to see if it's available or not.